recipes/cairo/meson/conanfile.py

```diff
@@ -86,21 +86,21 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("pixman/0.42.2")
+        self.requires("pixman/0.43.0")
         if self.options.with_zlib and self.options.with_png:
-            self.requires("expat/2.5.0")
+            self.requires("expat/2.6.0")
         if self.options.with_lzo:
             self.requires("lzo/2.10")
         if self.options.with_zlib:
             self.requires("zlib/[>=1.2.11 <2]")
         if self.options.with_freetype:
             self.requires("freetype/2.13.2", transitive_headers=True, transitive_libs=True)
         if self.options.with_fontconfig:
-            self.requires("fontconfig/2.14.2", transitive_headers=True, transitive_libs=True)
+            self.requires("fontconfig/2.15.0", transitive_headers=True, transitive_libs=True)
         if self.options.with_png:
-            self.requires("libpng/1.6.40")
+            self.requires("libpng/[>=1.6 <2]")
         if self.options.with_glib:
-            self.requires("glib/2.78.1")
+            self.requires("glib/2.78.3")
```

Comment:
```suggestion
            self.requires("glib/2.80.0")
```
It's not out yet, but it might be worth waiting for #23193 to be merged.

Reply:
the PR to bump glib has been closed, not merged.